packages/server/src/schema/events/resolvers.ts

```diff
@@ -22,15 +23,15 @@ export const resolvers: Resolvers = {
       const actorId = payload.actorId
         ? payload.actorId
         : actorUsername
-          ? await resolveIdFromUsername(actorUsername, context)
+          ? await resolveIdFromUsername(actorUsername, database)
```

Comment:
```suggestion
          ? await resolveIdFromUsername(actorUsername, context)
```

Please use the context here. This is the way we already did it in many `resolve()` functions in the code base and I would like to keep it consistent. This makes adding new context dependencies also easier. https://effect.website/ Also handles the context in this way.